test/extended/testdata/cmd/test/cmd/policy-storage-admin.sh

```diff
@@ -39,7 +39,7 @@ os::cmd::expect_failure_and_text 'oc policy --as=storage-adm remove-user screele
 #os::cmd::expect_success 'oc login -u storage-adm2 -p pw'
 #os::cmd::expect_success_and_text 'oc whoami' "storage-adm2"
 os::cmd::expect_success 'oc new-project --as=storage-adm2 --as-group=system:authenticated:oauth --as-group=system:authenticated policy-can-i'
-os::cmd::expect_success_and_text 'oc policy --as=storage-adm2 can-i create pod --all-namespaces' 'no'
+os::cmd::expect_failure_and_text 'oc policy --as=storage-adm2 can-i create pod --all-namespaces' 'no'
 os::cmd::expect_success_and_text 'oc policy --as=storage-adm2 can-i create pod' 'yes'
 os::cmd::expect_success_and_text 'oc policy --as=storage-adm2 can-i create pvc' 'yes'
 os::cmd::expect_success_and_text 'oc policy --as=storage-adm2 can-i create endpoints' 'yes'
```

Comment:
these should be 'oc auth can-i' also, right?

Reply:
Good catch!